src/ripple/ledger/View.h

```diff
@@ -110,12 +134,41 @@ forEachItem(
 bool
 forEachItemAfter(
     ReadView const& view,
-    AccountID const& id,
+    Keylet const& root,
     uint256 const& after,
     std::uint64_t const hint,
     unsigned int limit,
     std::function<bool(std::shared_ptr<SLE const> const&)> f);
 
+/** Iterate all items in an account's owner directory. */
+inline void
+forEachItem(
+    ReadView const& view,
+    AccountID const& id,
+    std::function<void(std::shared_ptr<SLE const> const&)> f)
+{
+    return forEachItem(view, keylet::ownerDir(id), std::move(f));
```

Comment:
No need to move once we change the parameter type to `const&` in the `forEach` functions.